Makefile

```diff
@@ -87,13 +87,3 @@ clean:
 	-@rm -f pyop2/*.so > /dev/null 2>&1
 	@echo "    RM tinyasm/*.so"
 	-@rm -f tinyasm/*.so > /dev/null 2>&1
-
-.PHONY: durations
-durations:
```

Comment:
Why get rid of this? Surely it's easier for someone to run `make durations` than to copy out that whole `mpiexec` command?

Reply:
It wasn't robust. I was getting some test failures locally because I didn't have the exact CI setup installed. Now I think about it I guess a better solution here is to run this command in the Docker image...

Converting to draft...